android/src/main/java/com/clevertap/clevertap_plugin/DartToNativePlatformCommunicator.kt

```diff
@@ -1974,7 +2004,7 @@ class DartToNativePlatformCommunicator(
             } else {
                 result.error(
                     TAG,
-                    ERROR_TEMPLATE_NAME,
+                    ERROR_TEMPLATE_NAME(templateName),
```

Comment:
Template name is repeated, might be redundant?

Reply:
added details as null since its uniformly passed as null in all places.